CHANGELOG.md

```diff
@@ -7,10 +7,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 [Unpublished]
 -------------
 ### Added
+- **New icons:** mruby(`.mrb`)
 - **Support:** Git (`.keep`), `.vagrant` folders
 
 ### Fixed
 - [[`#429`](https://github.com/DanBrooker/file-icons/issues/429)] Tab-icon colours not obeying value of user's _"Changed only"_ setting.
+- Fix tab contamination in index.coffee
```

Comment:
Please keep the change-log for **notable** changes only.

This is temporary anyway, because the entire `index.coffee` file is due to be replaced with V2, which features [an entire rewrite](https://github.com/DanBrooker/file-icons/blob/v2/lib/). So it's not worth including this. :)